CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+[...]
+
+# v7.0.1 (21/06/2019)
+
 - **[FIX]** Removed forwardProps and added `SIZE` static property on `Modal`
```

Comment:
Just saying, when removing something, it's generally a BREAKING CHANGE

Reply:
Yes you're right